archive/overlap.go

```diff
@@ -11,7 +11,7 @@ import (
 // spans, and returns an ordered list of spanInfos, whose spans will be bounded
 // by filter, and where each SpanInfo contains one or more Chunks whose data
 // falls into the SpanInfo's span.
-func mergeChunksToSpans(chunks []Chunk, dir zbuf.Direction, filter nano.Span) []SpanInfo {
+func mergeChunksToSpans(chunks []Chunk, dir zbuf.Order, filter nano.Span) []SpanInfo {
```

Comment:
Change the name of the dir parameter to order in all of these functions.
```suggestion
func mergeChunksToSpans(chunks []Chunk, order zbuf.Order, filter nano.Span) []SpanInfo {
```